src/Microsoft.SqlTools.ServiceLayer/TableDesigner/TableDesignerValidator.cs

```diff
@@ -392,7 +392,8 @@ public List<TableDesignerIssue> Run(Dac.TableDesigner designer)
             {
                 errors.Add(new TableDesignerIssue()
                 {
-                    Description = "System versioned table must have the period columns defined."
+                    Description = "System versioned table must have the period columns defined.",
+                    MoreInfoLink = "https://docs.microsoft.com/sql/relational-databases/tables/creating-a-system-versioned-temporal-table?view=sql-server-ver15"
```

Comment:
should this URL contain a parameter for a specific SQL Server version, such as `sql-server-ver15`?  If so, should it come from the version the user is connected to?  Perhaps if the parameter is omitted MSDN will auto-navigate to the latest version, which is probably fine.

Reply:
+1 - remove that query parameter. Same as the LCIDs we should let the server handle redirecting to the appropriate final link for general links like this.